src/main/java/org/folio/rest/impl/UserImportAPI.java

```diff
@@ -465,6 +483,37 @@ private ImportResponse processSuccessfulImportResponse(List<Future> futures) {
     return future.future();
   }
 
+  private Future<RequestPreference> createUserPreference(User user, UserImportData userImportData,
+                                                         Map<String, String> okapiHeaders) {
+
+    RequestPreference requestPreference = userImportData.getRequestPreference().get(user.getUsername());
+    if (Objects.nonNull(requestPreference)) {
+      requestPreference.setUserId(user.getId());
```

Comment:
Does id in `user` update after creation of user if in request was user without id?